metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/InfluxDbReporter.java

```diff
@@ -136,10 +138,22 @@ public Builder includeMeterFields(Set<String> fields) {
             return this;
         }
 
+        /**
+         * Map measurement to a defined measurement name, where the key is the measurement name
+         * and the value is the reqex the measurement should be mapped by.
+         *
+         * @param measurementMappings
+         * @return {@code this}
+         */
+        public Builder measurementMappings(Map<String, Pattern> measurementMappings) {
```

Comment:
Cool, though I think from a usability stand-point we should pass in `Map<String, String>` and compile the patterns in this method.